terraform/modules/livy-proxy/alb.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_lb" "lb" {
+  name                             = "${var.name}-lb"
```

Comment:
Is this name unique enough? Could we be overriding existing logs in the ELBLog directory for another load balancer?

Reply:
It's what we do for all our load balancers (and other resources), `var.name` is `livy-proxy` and we do not have any resources that use that name prefix.